backend/src/libs/types/service.type.ts

```diff
@@ -1,10 +1,10 @@
 type Service<T = unknown> = {
-  find(): Promise<T>;
+  find(id: number): Promise<T>;
   findAll(): Promise<{
     items: T[];
   }>;
-  create(payload: unknown): Promise<T>;
-  update(): Promise<T>;
+  create(payload: unknown, id?: number): Promise<T>;
+  update(id: number, userId: number, payload: unknown): Promise<T>;
```

Comment:
I guess these params are not so generic. Maybe let's use one payload as an object to pass needed params?